resources/lang/en/beatmap_discussions.php

```diff
@@ -26,6 +26,7 @@
             'deleted' => 'Include deleted discussions',
             'mode' => 'Beatmap mode',
             'only_unresolved' => 'Show only unresolved discussions',
+            'show_review_embeds' => 'Show review posts',
```

Comment:
Is it posts or embeds? `Review posts` also implies the actual review post, not the posts in it 🤔

Reply:
yeah but I don't know if embed means anything to people